mozci/task.py

```diff
@@ -451,6 +452,24 @@ def failing_tasks(self):
             )
         ]
 
+    @memoized_property
+    def is_config_consistent_failure(self) -> Optional[bool]:
+        config_to_results = collections.defaultdict(list)
+        for task in self.tasks:
+            for result in task.results:
+                if result.group == self.name:
+                    config_to_results[task.configuration].append(result.ok)
```

Comment:
Unrelated, but this reminded me we still haven't made use of the `test-setting` dict here.

Reply:
Indeed, #557. We should do it soon.